satpy/etc/enhancements/generic.yaml

```diff
@@ -1230,3 +1230,60 @@ enhancements:
          stretch: crude
          min_stretch: [0,0,0]
          max_stretch: [1,1,1]
+
+  Cloud_Top_Height:
+    standard_name: cldprop_cloud_top_height
+    operations:
+      - name: colorize
+        method: !!python/name:satpy.enhancements.colorize
+        kwargs:
+          palettes:
+          - {colors: [[255, 0, 0], [255, 0, 0]], min_value: 0, max_value: 800}
+          - {colors: [[170, 0, 0], [170, 0, 0]], min_value: 800.0001, max_value: 1600}
+          - {colors: [[110, 0, 0], [110, 0, 0]], min_value: 1600.0001, max_value: 2350}
+          - {colors: [[112, 1, 2], [112, 1, 2]], min_value: 2350.0001, max_value: 3150}
+          - {colors: [[124, 91, 5], [124, 91, 5]], min_value: 3150.0001, max_value: 4000}
+          - {colors: [[240, 190, 64], [240, 190, 64]], min_value: 4000.0001, max_value: 4800}
+          - {colors: [[255, 255, 0], [255, 255, 0]], min_value: 4800.0001, max_value: 5600}
+          - {colors: [[0, 220, 0], [0, 220, 0]], min_value: 5600.0001, max_value: 6400}
+          - {colors: [[0, 136, 0], [0, 136, 0]], min_value: 6400.0001, max_value: 7200}
+          - {colors: [[0, 80, 0], [0, 80, 0]], min_value: 7200.0001, max_value: 8000}
+          - {colors: [[0, 136, 238], [0, 136, 238]], min_value: 8000.0001, max_value: 8800}
+          - {colors: [[0, 0, 255], [0, 0, 255]], min_value: 8800.0001, max_value: 9600}
+          - {colors: [[0, 0, 170], [0, 0, 170]], min_value: 9600.0001, max_value: 10400}
+          - {colors: [[0, 0, 100], [0, 0, 100]], min_value: 10400.0001, max_value: 11200}
+          - {colors: [[183,15,141], [183, 15, 141]], min_value: 11200.0001, max_value: 12000}
+          - {colors: [[102, 0, 119], [102, 0, 119]], min_value: 12000.0001, max_value: 18000}
```

Comment:
You should be able to define this as two parameters: colors and values. This should be much faster for Satpy to load this information in. As you have it Satpy/trollimage is considering each of these as a separate colormap.

Reply:
Why I did it like this is because when I just had values and colors it interpolated between the colors like we were talking about before.  The only way I could get it to not interpolate was to have satpy see it as separate colormaps with those defined values.

Reply:
I reverted back to just having the two values. The images attached show the issue
<img width="1248" alt="Screenshot 2024-02-09 at 1 41 31 PM" src="https://github.com/pytroll/satpy/assets/153020597/21a5473c-9c1d-4a6a-b59e-4984c4cc35b5">
<img width="1248" alt="Screenshot 2024-02-09 at 1 43 26 PM" src="https://github.com/pytroll/satpy/assets/153020597/c12398fd-1e72-4619-a5e1-7ed9589c3c0a">

Reply:
Unless github mobile is showing it wrong, you still need to double every color and have the values as you had them. I'll check more closely when I'm at my computer later.

Reply:
No, I think you're right. I can make those changes and then put it in a csv in a few minutes

Reply:
should be all fixed now